pkg/bootstrap/config.go

```diff
@@ -402,6 +402,15 @@ func extractAttributesMetadata(envVars []string, plat platform.Environment, meta
 	}
 	if plat != nil && len(plat.Metadata()) > 0 {
 		meta.PlatformMetadata = plat.Metadata()
+		// unlike k8s deployments where owner info is populated from the injector,
+		// vms must populate owner info from the platform metadata.
+		// TODO: with the advent of WorkloadEntry, this might need to be expanded
+		// to (a) be more general and (b) support workload name and labels extraction
+		if len(meta.Owner) == 0 {
+			// attempt to populate with GCE Owner information. If not a GCE instance,
```

Comment:
Nevermind I misunderstood this a bit. It does seem like this should be `plat.GetOwner()` though or something similar?

Reply:
Yeah, I thought about this for a moment... and didn't come up with a great way to do it without more refactoring than I was excited about.

Adding `plat.GetOwner()` would mean updating that method with owner logic from any platform.

Maybe it would be better to just have the platforms return metadata with an owner field that can be used to override.  That's fairly simple, and can be generic.